librz/debug/trace.c

```diff
@@ -51,20 +51,20 @@ RZ_API bool rz_debug_trace_ins_before(RzDebug *dbg) {
 	// Analyze current instruction
 	ut64 pc = rz_debug_reg_get(dbg, dbg->reg->name[RZ_REG_NAME_PC]);
 	if (!dbg->iob.read_at) {
-		eprintf("Error: dbg->iob.read_at missing\n");
+		RZ_LOG_ERROR("Error: dbg->iob.read_at missing\n");
```

Comment:
This will output `Error: Error:  dbg->iob.read_at missing`

Reply:
same for the others below

Reply:
oh sorry c89b4c49914a50ed4fad91124cc9a9675c9a1b5a